gr-uhd/CMakeLists.txt

```diff
@@ -37,6 +42,7 @@ GR_REGISTER_COMPONENT("gr-uhd" ENABLE_GR_UHD
     ENABLE_GR_ANALOG
 )
 message(STATUS "  UHD Version: ${UHD_VERSION}")
+message(STATUS "  UHD 4.0 RFNoC enabled: ${ENABLE_UHD_RFNOC}")
 
 SET(GR_PKG_UHD_EXAMPLES_DIR ${GR_PKG_DATA_DIR}/examples/uhd)
```

Comment:
Likewise, please move this variable setting into the `if(ENABLE_GR_UHD)` below, since it's not needed unless UHD is enabled.